core/api/src/graphql/public/types/payload/quiz-completed.ts

```diff
@@ -3,6 +3,7 @@ import Quiz from "../object/quiz"
 
 import { GT } from "@/graphql/index"
 
+// deprecated
```

Comment:
if this is deprecated add `deprecationReason` to the query/mutation

Reply:
it's a payload, we can only add `deprecationReason` to the mutation, and this has already be added https://github.com/GaloyMoney/galoy/pull/3743/files#diff-dc640d972fe6a4a4fc5e94759ee942cf9d4940224c264530eb012ebacccbff1bR1172